README.md

```diff
@@ -71,30 +71,73 @@ make install
 ```
 
 ### Windows platform ###
+**note**: *make sure the following compile tools or libraries has been installed before install dependency libraries*
+
+- compile tools:  **CMake**、**Visusal Studio 2013**
+
+- libraries:   **zlib**

```

Comment:
Can you explain the specific version information of zlib?

Reply:
it was specified at here, and link was added 
3. install boost 1.56.0 and zlib 1.2.3